packages/web-app-files/src/mixins/spaces/actions/showDetails.js

```diff
@@ -10,17 +10,24 @@ export default {
           iconFillType: 'line',
           label: () => this.$gettext('Details'),
           handler: this.$_showDetails_trigger,
-          isEnabled: () => false, // @TODO As soon as we have the details
+          isEnabled: ({ resources }) => resources.length === 1,
           componentType: 'oc-button',
           class: 'oc-files-actions-show-details-trigger'
         }
       ]
     }
   },
   methods: {
-    ...mapActions('Files/sidebar', { openSidebar: 'open' }),
+    ...mapActions('Files/sidebar', { openSidebar: 'open', closeSidebar: 'close' }),
+    ...mapMutations('Files', ['SET_FILE_SELECTION']),
 
-    async $_showDetails_trigger() {
+    async $_showDetails_trigger({ resources }) {
+      if (resources.length !== 1) {
+        return
+      }
+
+      this.SET_FILE_SELECTION([resources[0]])
```

Comment:
These two added lines look like a workaround / something didn't work out as planned. What are you trying to solve? Why do you need to close the sidebar before opening it again?

Reply:
The way `showDetails` is invoked from files related contexts already does the selection beforehand (only appears in the context menu. showing the context menu already sets the selected resource). Is that not the case for showing space details?